doc/rst/usingchapel/prereqs.rst

```diff
@@ -186,13 +191,15 @@ We have used the following commands to install the above prerequisites:
 Compatibility Notes
 -------------------
 
-Amazon Linux 2 CHPL_LLVM!=system incompatibility
+Amazon Linux 2 CHPL_LLVM==system incompatibility
 ++++++++++++++++++++++++++++++++++++++++++++++++
 
 Amazon Linux 2 uses GCC 7.3.1 but GCC 7.4 or newer is required to build
-LLVM. Chapel can use a system-wide install of LLVM on this platform, so
-installing the LLVM packages as shown above and using `CHPL_LLVM=system`
-is the recommended workaround.
+LLVM, and also uses LLVM 11 but LLVM 14 or newer is required for Chapel.
+To use Chapel on this platform, installing a newer GCC is required. The repos
+provide a GCC 10 package, which we then set environment variables for Chapel to
+use instead of the default version. Chapel can then be built with
+``CHPL_LLVM=none`` or ``CHPL_LLVM=bundled``.
```

Comment:
I think this can be cleaned up a bit. Consider:

> Amazon Linux 2 uses GCC 7.3.1 and only provides LLVM 11, but Chapel requires a newer GCC and newer LLVM. To use Chapel on this platform, installing a newer GCC is required. The repositories provide a GCC 10 package, which can be used to configure Chapel.
>
>  ```
>  export CC=gcc10-gcc
>  export CXX=gcc10-g++
>  export CHPL_HOST_CC=gcc10-gcc
>  export CHPL_HOST_CXX=gcc10-g++
>  ```
>
>  Chapel can then be built with either ``CHPL_LLVM=none`` (still requires a newer GCC) or ``CHPL_LLVM=bundled``